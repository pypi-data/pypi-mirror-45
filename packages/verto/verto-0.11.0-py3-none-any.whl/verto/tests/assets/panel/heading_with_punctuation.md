{panel type="note"}

# crwdns8254:0crwdne8254:0

crwdns8255:0crwdne8255:0 crwdns8256:0crwdne8256:0 crwdns8257:0crwdne8257:0 crwdns8258:0crwdne8258:0

{panel end}
