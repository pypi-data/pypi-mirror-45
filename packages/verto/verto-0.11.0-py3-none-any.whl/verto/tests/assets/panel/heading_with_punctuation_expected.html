<div class="panel panel-note">
<div class="panel-header">
<strong>crwdns8254:0crwdne8254:0</strong>
</div>
<div class="panel-body">
<p>crwdns8255:0crwdne8255:0 crwdns8256:0crwdne8256:0 crwdns8257:0crwdne8257:0 crwdns8258:0crwdne8258:0</p>
</div>
</div>
